libbeat/filter/config.go

```diff
@@ -0,0 +1,14 @@
+package filter
+
+type DropFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type IncludeFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type FilterConfig struct {
+	DropFields    *DropFieldsConfig    `config:"drop_fields"`
```

Comment:
Makes sense. Would it be perhaps `SelectFields` instead of `IncludFields` if we apply the same logic?